container-test/pom.xml

```diff
@@ -30,12 +30,20 @@
           <groupId>com.yahoo.vespa</groupId>
           <artifactId>vespajlib</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcpkix-jdk15on</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
 
     <!-- All dependencies that should be visible in test classpath, but not compile classpath,
          for user projects must be added in compile scope here. These dependencies are explicitly excluded
          (or set to non-compile scope) in the container and/or container-dev modules. -->
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+    </dependency>
     <dependency>
```

Comment:
Suggest to move this above the multi-line comment, and add a separate comment saying that this is added back for the exclusion above to work around a presumed bug in maven dep res.